docs/quickstart.rst

```diff
@@ -2,54 +2,17 @@
 Quick start
 =============
 
-Example locustfile.py
-=====================
+In Locust you define your user behaviour in plain Python code. You then use the ``locust`` command and (optionally) its web interface to spawn and simulate a number of those users while gathering request statistics.
 
-When using Locust you define the behaviour of users in Python code, and then you have the ability to 
-simulate any number of those users while gathering request statistic. The entrypoint for defining the 
-user behaviour is the `locustfile.py`.
-
-.. note::
-
-    The ``locustfile.py`` is a normal Python file that will get imported by Locust. Within it you 
-    can import modules just as you would in any python code.
-    
-    The file can be named something else and specified with the `-f` flag to the ``locust`` command.
-
-Below is a quick little example of a simple **locustfile.py**:
-
-.. code-block:: python
-    
-    import random
-    from locust import HttpUser, task, between
-    
-    class WebsiteUser(HttpUser):
-        wait_time = between(5, 9)
-        
-        @task(2)
-        def index(self):
-            self.client.get("/")
-            self.client.get("/ajax-notifications/")
-        
-        @task(1)
-        def view_post(self):
-            post_id = random.randint(1, 10000)
-            self.client.get("/post?id=%i" % post_id, name="/post?id=[post-id]")
-        
-        def on_start(self):
-            """ on_start is called when a User starts before any task is scheduled """
-            self.login()
+locustfile.py
+=============
+.. literalinclude:: ../examples/locustfile.py
```

Comment:
I think it's better to keep the example inline. Since we're "breaking it down" immediately below it'll make it slitghtly easier to maintain this documentation file (and not risking that we accidentally do some changes to `../examples/locustfile.py` without updating quickstart.rst at some point in the future).

Reply:
I just realised that in this PR the `locusfile.py` and the "break down" is already out of sync. The first task is called `index` in the file and `hello` in the break down.

Reply:
I split it out to be able to test run it exactly as it is. I almost commited multiple out of sync things before moving it to a separate file, so it doesnt really help :)

What *would* help (and I think improve readability) is if we just had a locustfile with inline comments instead. How do you feel about that?

Reply:
> I split it out to be able to test run it exactly as it is. I almost commited multiple out of sync things before moving it to a separate file, so it only helps marginally :)

Ah, I see! What I did when I wrote it initially was that that I just used a temporary file to test it, and then copy it back into the quickstart.rst.

> What would help (and I think improve readability) is if we just had a locustfile with inline comments instead. How do you feel about that?

Hmm, I'm not sure. I think the current format allow us to easier highlight what part of the code the comment is referring to E.g:
![image](https://user-images.githubusercontent.com/54217/82320759-199c1f80-99d4-11ea-9771-ba307515ddc5.png)

It would definitely be easier to maintain a file with comments, but I do think it's worth the effort to use the current format (since it's not something that should change super frequently).

Reply:
Ok, I'll add it back inline, no biggie.